tests/test_rigid_physics.py

```diff
@@ -2378,3 +2387,51 @@ def test_mesh_to_heightfield(show_viewer):
     # speed is around 0
     qvel = ball.get_dofs_velocity().cpu()
     assert_allclose(qvel, 0, atol=1e-2)
+
+
+@pytest.mark.mujoco_compatibility(True)
+@pytest.mark.gjk_collision(True)
+@pytest.mark.multi_contact(False)
+@pytest.mark.parametrize("xml_path", ["xml/tet_tet.xml"])
+@pytest.mark.parametrize("gs_solver", [gs.constraint_solver.CG, gs.constraint_solver.Newton])
+@pytest.mark.parametrize("gs_integrator", [gs.integrator.implicitfast, gs.integrator.Euler])
+@pytest.mark.parametrize("backend", [gs.cpu])
+def test_tet_tet(gs_sim, mj_sim, gs_solver, tol):
+    # Make sure it is possible to set the configuration vector without failure
+    gs_sim.rigid_solver.set_dofs_position(gs_sim.rigid_solver.get_dofs_position())
+
+    check_mujoco_model_consistency(gs_sim, mj_sim, tol=tol)
+    tol = 1e-6
```

Comment:
Why using `tol` does not work? Using a custom tolerance for matching mujoco should be very well motivated. If this is necessary, add a comment to explain why it cannot do better.

Reply:
Let's keep this issue for later until every other comments are resolved.